bus/bus.go

```diff
@@ -1633,15 +1644,16 @@ func (b *bus) Handler() http.Handler {
 		"POST   /wallet/prepare/renew": b.walletPrepareRenewHandler,
 		"GET    /wallet/pending":       b.walletPendingHandler,
 
-		"GET    /hosts":              b.hostsHandlerGET,
-		"GET    /host/:hostkey":      b.hostsPubkeyHandlerGET,
-		"POST   /hosts/interactions": b.hostsPubkeyHandlerPOST,
-		"POST   /hosts/remove":       b.hostsRemoveHandlerPOST,
-		"GET    /hosts/allowlist":    b.hostsAllowlistHandlerGET,
-		"PUT    /hosts/allowlist":    b.hostsAllowlistHandlerPUT,
-		"GET    /hosts/blocklist":    b.hostsBlocklistHandlerGET,
-		"PUT    /hosts/blocklist":    b.hostsBlocklistHandlerPUT,
-		"GET    /hosts/scanning":     b.hostsScanningHandlerGET,
+		"GET    /hosts":                   b.hostsHandlerGET,
+		"GET    /host/:hostkey":           b.hostsPubkeyHandlerGET,
+		"POST   /hosts/scans":             b.hostsScanHandlerPOST,
+		"POST   /hosts/pricetableupdates": b.hostsPricetableHandlerPOST,
```

Comment:
I know it's a bit of a `nit` but it's now or never: should we rename this and drop "updates" so it becomes `hosts/pricetables`